src/cli.ts

```diff
@@ -1,51 +1,537 @@
+      deprecated: true,
+      description: 'path to Chrome flash plugin; find it in `chrome://plugins`',
+      group: 'Flash Options (DEPRECATED)',
+      normalize: true,
+      type: 'string',
+    })
+    // Platform Specific Options
+    .option('app-copyright', {
+      description:
+        '(macOS, windows only) a human-readable copyright line for the app. Maps to `LegalCopyright` metadata property on Windows, and `NSHumanReadableCopyright` on macOS',
+      group: 'Platform Specific Options',
+      type: 'string',
+    })
+    .option('app-version', {
+      description:
+        '(macOS, windows only) the version of the app. Maps to the `ProductVersion` metadata property on Windows, and `CFBundleShortVersionString` on macOS.',
+      group: 'Platform Specific Options',
+      type: 'string',
+    })
+    .option('bounce', {
+      default: false,
+      description:
+        '(macOS only) make the dock icon bounce when the counter increases',
+      group: 'Platform Specific Options',
+      type: 'boolean',
+    })
+    .option('build-version', {
+      description:
+        '(macOS, windows only) The build version of the app. Maps to `FileVersion` metadata property on Windows, and `CFBundleVersion` on macOS',
+      group: 'Platform Specific Options',
+      type: 'string',
+    })
+    .option('counter', {
+      default: false,
+      description:
+        '(macOS only) set a dock count badge, determined by looking for a number in the window title',
+      group: 'Platform Specific Options',
+      type: 'boolean',
+    })
+    .option('darwin-dark-mode-support', {
+      default: false,
+      description: '(macOS only) enable Dark Mode support on macOS 10.14+',
+      group: 'Platform Specific Options',
+      type: 'boolean',
+    })
+    .option('f', {
+      alias: 'fast-quit',
+      default: false,
+      description: '(macOS only) quit app on window close',
+      group: 'Platform Specific Options',
+      type: 'boolean',
+    })
+    .option('title-bar-style', {
+      choices: ['hidden', 'hiddenInset'],
+      description:
+        '(macOS only) set title bar style. Consider injecting custom CSS (via --inject) for better integration',
+      group: 'Platform Specific Options',
+      type: 'string',
+    })
+    .option('win32metadata', {
+      coerce: parseJson,
+      description:
+        '(windows only) a JSON string of key/value pairs (ProductName, InternalName, FileDescription) to embed as executable metadata',
+      group: 'Platform Specific Options',
+      type: 'string',
+    })
+    // Debug Options
+    .option('crash-reporter', {
+      description: 'remote server URL to send crash reports',
+      group: 'Debug Options',
+      type: 'string',
+    })
+    .option('verbose', {
+      default: false,
+      description: 'enable verbose/debug/troubleshooting logs',
+      group: 'Debug Options',
+      type: 'boolean',
+    })
+    .version()
+    .help()
+    .group(['version', 'help'], 'Other Options')
```

Comment:
Nit, maybe internal to yargs and not a big deal if not polishable: why do these show up as `[boolean]` in the final result?

```
Other Options
      --version  Show version number                 [boolean]
      --help     Show help                           [boolean]
                                                THIS ^^^^^^^^^
```

Reply:
Because they technically are, as in is it true you want to show the help/version?

I'll see what I can do about dropping it, but I think that's the only way to have a flag that doesn't take a value. (i.e., `--help` not `--help <value>`

Reply:
Alright. But again, it's a super nit and not a big deal. If this is what it is, no problem.